### DISCUSSION TITLE
How can i use Zustand instead of MobX-State-Tree

### DISCUSSION THREAD

**Discussion by otekdo** [infinitered/ignite#2277] [Q&A] [answered]
Hello, have anyone use here ``zustand`` with ignite? 
I would like to know how is possible to use ``zustand`` primarily instead of ``MST`` for this awesome boilerplate
https://github.com/pmndrs/zustand

They even have persistence middleware
https://github.com/pmndrs/zustand/blob/main/docs/integrations/persisting-store-data.md

**Comment by frankcalise** [comment, marked as answer]
@otekdo thanks for the idea! I invite you to make a recipe request over at https://github.com/infinitered/ignite-cookbook. This is a repo where we provide walkthroughs on features/enhancements/changes to Ignite based on other packages that we may or may not use on our client projects.
